app/components/layout/project/project.controller.js

```diff
@@ -0,0 +1,35 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('app.layout')
+    .controller(
+      'ProjectController',
+      ProjectController
+      );
+
+  /* @ngInject */
+  function ProjectController(ProjectService, ngNotify) {
+    var vm = this;
+    vm.projects = [];
+    vm.datapoints = [];
+    vm.datacolumns = [{'id': 'top-1', 'type': 'bar', 'name': 'Projects'}];
+    vm.datax = {'id': 'x'};
+
+    getProjects();
+
+    function getProjects() {
+      var tmpList = [];
+      ProjectService.getProjects()
+        .then(function(projectsData) {
+          vm.projects = projectsData.data;
+          _.forEach(projectsData.data, function(value, key) {
+            vm.datapoints.push({'x': value.name, 'top-1': value.percentage});
+          });
+        })
+        .catch(function(error) {
+          ngNotify.set('Error loading projects', 'error');
+        });
```

Comment:
From what I have seen this function in many controllers. Maybe you could create a service to show the error message and just call it here. (?) Just a suggestion.